client/me/purchases/cancel-purchase/refund-information.jsx

```diff
@@ -294,10 +294,23 @@ const CancelPurchaseRefundInformation = ( {
 				},
 			}
 		);
+	} else if ( maybeWithinRefundPeriod( purchase ) ) {
+		text = i18n.translate(
+			"Clicking the Cancel Subscription button below will disable auto-renew and you'll be able to use %(productName)s " +
+				'until your subscription expires. When your subscription expires, the paid plan features will be automatically ' +
+				'removed from your site.',
+			{
+				args: {
+					productName: getName( purchase ),
+				},
+			}
+		);
 	} else {
 		text = i18n.translate(
-			"When you cancel your subscription, you'll be able to use %(productName)s until your subscription expires. " +
-				'Once it expires, it will be automatically removed from your site.',
+			'Your purchase is outside of the money-back guarantee window and is not eligible for a refund. ' +
+				"Clicking the Cancel Subscription button below will disable auto-renew and you'll be able to use %(productName)s " +
+				'until your subscription expires. When your subscription expires, the paid plan features will be automatically ' +
+				'removed from your site.',
```

Comment:
Similar to an earlier comment, this text is used for more than just plans (for example, Jetpack Search) so you have to be careful adding plan-specific language to it.